spec/controllers/projects/contributions_controller_spec.rb

```diff
@@ -110,6 +124,62 @@
     end
   end
 
+  describe 'project.online?' do
```

Comment:
It's better you do something like:

```ruby
describe 'project' do
  context 'when the state is different of online' do
    # ...
  end

  context 'when the state is online' do
    # ...
  end

  context 'when the project is unavailable for contributions' do
    # ...
  end

  context 'when the project is available for contributions' do
    # ...
  end
end
```

This way you can share some elements like:
```ruby
let(:user){ create(:user).decorate }

before do
  allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
end
```